edge-patron/src/main/resources/karate-config.js

```diff
@@ -0,0 +1,84 @@
+function fn() {
+
+  karate.configure('logPrettyRequest', true);
+  var retryConfig = { count: 20, interval: 30000 }
+  karate.configure('retry', retryConfig)
+
+  var env = karate.env;
+
+  // The "testTenant" property could be specified during test runs
+  var testTenant = karate.properties['testTenant'];
+
+  var config = {
+    baseUrl: 'http://localhost:9130',
+    admin: {tenant: 'diku', name: 'diku_admin', password: 'admin'},
+
+    testTenant: testTenant ? testTenant: 'testTenant',
+    testAdmin: {tenant: testTenant, name: 'test-admin', password: 'admin'},
+    testUser: {tenant: testTenant, name: 'test-user', password: 'test'},
+
+    // define global features
+    login: karate.read('classpath:common/login.feature'),
+    dev: karate.read('classpath:common/dev.feature'),
+
+    // define global functions
+    uuid: function () {
+      return java.util.UUID.randomUUID() + ''
+    },
+
+    random: function (max) {
+      return Math.floor(Math.random() * max)
+    },
+
+    randomMillis: function() {
+      return java.lang.System.currentTimeMillis() + '';
+    },
+
+    random_string: function() {
+      var text = "";
+      var possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
+      for (var i = 0; i < 5; i++)
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
+      return text;
+    },
+
+    //to generate random barcode
+    random_numbers: function() {
+      var text = Math.floor(Math.random() * 1000000);
+      return text;
+    }
+  };
+
+  // Create 100 functions for uuid generation
+  var rand = function(i) {
+    karate.set("uuid"+i, function() {
+      return java.util.UUID.randomUUID() + '';
+    });
+  }
+  karate.repeat(100, rand);
```

Comment:
Why are a hundred functions needed? Does Karate evaluate the new UUID when the function is defined not when it is executed?

Reply:
This is similar in all projects and i have copied it from there , i will correct it in next commit. Thankyou.